src/main/java/com/opentable/versionedconfig/VersionedConfig.java

```diff
@@ -29,21 +28,10 @@
     static final String X_OAUTH_BASIC = "x-oauth-basic";
 
     @Bean
-    public GitProperties defaultVersioningServiceProperties(@Value("${config.repo.remote}") URI remoteRepo,
-                                                            @Value("${config.repo.oauth-token:#{null}}") String token,
-                                                            @Value("${config.repo.username:#{null}}") String username,
-                                                            @Value("${config.repo.password:#{null}}") String password,
+    public GitProperties defaultVersioningServiceProperties(@Value("${config.repo.remote}") List<URI> remoteRepo,
```

Comment:
Rather than relying on method name, shouldn't this be @Named just like the consuming bean.

@Named(DEFAULT_VERSIONING) for both?

Reply:
Huh?  I don't understand this.  Where is the bean name important?

Reply:
Look at the whole file.